src/schema/artwork/__tests__/meta.test.ts

```diff
@@ -0,0 +1,77 @@
+import { runQuery } from "test/utils"
+
+describe("Meta", () => {
+  const artworkData = {
+    artists: [
+      {
+        name: "Hans Hartung",
+      },
+    ],
+    title: "P. 25-1975-H-8",
+    date: "1975",
+    forsale: true,
+    dimensions: {
+      in: "22 4/5 × 30 7/10 in",
+      cm: "58 × 78 cm",
+    },
+    medium: "Acrylic on baryte card",
+    metric: "cm",
+    partner: {
+      name: "Galerie Michel Descours",
+    },
+  }
+
+  const rootValue = {
+    artworkLoader: () => Promise.resolve(artworkData),
+  }
+
+  describe("#description", () => {
+    it("returns properly formatted string", async () => {
+      const query = `
+        {
+          artwork(id:"hans-hartung-p-25-1975-h-8") {
+            meta {
+              description
+            }
+          }
+        }
+      `
+
+      const data = await runQuery(query, rootValue as any)
```

Comment:
@alloy - I found this `as any` kinda weird, and couldn't seem to reproduce it from other `.ts` test files. Thoughts? It seems to be picking up the default argument from `runQuery` (`{ userID, accessToken }`) and assuming that's its interface.